cpp/tests/CMakeLists.txt

```
@@ -131,16 +143,11 @@ if(RAPIDSMPF_HAVE_MPI)
     disable_sign_conversion_warning(ucxx_tests)
 
     rapidsmpf_mpirun_test_add(TEST_TARGET ucxx_tests)
-  endif()
-endif()
+  endif(RAPIDSMPF_HAVE_UCXX)
+endif(RAPIDSMPF_HAVE_MPI)
```

Please put this back the way it was. CMake has not required putting the condition in the `endif()` for well over a decade, and it is generally discouraged to do so.

```suggestion
  endif()
endif()
```